lib/mix/tasks/compile.make.ex

```diff
@@ -164,4 +169,12 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   defp args_for_makefile("nmake", makefile), do: ["/F", makefile]
   defp args_for_makefile(_, :default), do: []
   defp args_for_makefile(_, makefile), do: ["-f", makefile]
+
+  defp print_verbose_info(exec, args) do
+    args = Enum.map_join(args, " ", fn(arg) ->
+      if String.contains?(arg, " "), do: inspect(arg), else: arg
+    end)
+
+    Mix.shell.info "Make command: #{exec} #{args}"
```

Comment:
Maybe "Compiling with make: #{exec} #{args}" would be more inline with what the Mix compilers prints today? Other than that, ship it. :D